lib/email_downcaser.rb

```diff
@@ -0,0 +1,8 @@
+class EmailDowncaser
+  def self.dump(value)
+    value.downcase
+  end
+  def self.load(value)
+    value
+  end
+end
```

Comment:
The bit in-front of the domain can be case sensitive, but I suspect it's very unlikely that it ever is.

I think it's to check against me using MURRAY.STEELE @ my domain and murray.steele @ my domain so I'd leave it as-is.  It's also likely that when we were storing the email address rather than the encrypted version we would not have been doing things in mysql to mean our "you've already signed this petition" lookups would have been case-sensitive, so downcasing the whole email address keeps that behaviour.